protocol/src/main/java/org/fidoalliance/fdo/protocol/serialization/ManufacturingInfoDeserializer.java

```diff
@@ -55,10 +55,16 @@ public boolean isValidString(String deviceString) {
   public ManufacturingInfo deserialize(JsonParser jp, DeserializationContext ctxt)
       throws IOException {
     JsonNode node = jp.getCodec().readTree(jp);
+    
+    int validMessageSize = 5;
```

Comment:
can we change this "5" to be a configurable variable instead of hardcoding here?